src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.hpp

```diff
@@ -607,6 +607,7 @@ struct OnnxToKrnlLoweringConfiguration {
   static int reportOnSimd;
   static std::string defaultSimdComment;
   static EnableByRegexOption enableSpecificParallelOps;
+  static EnableByRegexOption traceSpecificOpSignatures;
```

Comment:
It looks this variable should be inside InstrumentONNXSignaturePass. Also, this pass does not use it.

Reply:
I tried to move it inside but ran into issue. Let me try again, as I agree it would be the better solution.

Reply:
I found a better way to implement your recommendation than what I did before. Now it's included, no global state anymore. thanks for the suggestion, it was the right call.

Reply:
Thank you!